eng/fv/tool-setup.rst

```diff
@@ -0,0 +1,239 @@
+.. SPDX-License-Identifier: CC-BY-SA-4.0
+
+.. Copyright (C) 2022 Trinity College Dublin
+
+.. _FormalToolSetup:
+
+Formal Tools Setup
+==================
+
+A key requirement for methods and tools for RTEMS qualification was that they
+comply to a large degree with RTEMS community guidelines. We wanted to avoid
+having large complex tools with large library dependencies, and have tools and
+notations that would be easy to maintain.
+
+After reviewing a range of methods and tools, we settled on a technique based on
+the SPIN model checker :cite:`Holzmann:1997:SPIN`, with its modelling language
+called Promela (https://spinroot.com/). Promela is quite a low-level modelling
+language that makes it easy to get close to code level, and is specifically 
+targeted to modelling software. It is one of the most widely used 
+model-checkers, both in industry and education. 
+It uses linear-time temporal logic (:term:`LTL`) to express properties of interest.
+
+It is open-source (https://github.com/nimble-code/Spin), actively maintained,
+and very easy to install, needing only  a C compiler and the ``lex``/``yacc``
+utilities.
```

Comment:
this is repetitive to some text in the Promela section.

Reply:
ok - I'll remove one - perhaps from the Promela section